### DISCUSSION TITLE
Versioning of API in APIList page

### DISCUSSION THREAD

**Discussion by shashiguru** [Azure/api-management-developer-portal#2549] [Feature request]
## Bug description

Hi there, we have created 2 versions for an API. in apilist page, its showing as 2 different api's. instead of that if we can show only original API and if we can have versioning in api details page then that would be looks good as per functionality. 

##Current behavior
 API List page:
![image](https://user-images.githubusercontent.com/30490543/91169109-5c1c7900-e6f4-11ea-87e0-55aba7efa4f8.png)

API details page:
![image](https://user-images.githubusercontent.com/30490543/91169491-17451200-e6f5-11ea-9914-e8105e7616af.png)

## Expected behavior

API List page:
![image](https://user-images.githubusercontent.com/30490543/91169364-d947ee00-e6f4-11ea-8755-6a2f83cc30db.png)

API Details page:
![image](https://user-images.githubusercontent.com/30490543/91169550-2deb6900-e6f5-11ea-8e4c-4f40ea5219e1.png)

## Is your portal managed or self-hosted?

Managed

**Comment by azaslonov** [comment]
Hi @shashiguru, makes sense, I think we can add config to switch into grouping mode.

**Comment by mikebudzynski** [comment]
We had this design in the beginning but we changed it to the current version as per customer requests. We may introduce a widget configuration option to control this behavior.

**Comment by shashiguru** [comment]
Makes sense, thanks @mikebudzynski. when can we expect these changes?
